src/SonarScanner.MSBuild.Shim/AdditionalFilesService.cs

```diff
@@ -35,6 +35,12 @@ public class AdditionalFilesService(IDirectoryWrapper directoryWrapper) : IAddit
 {
     private static readonly char[] Comma = [','];
 
+    private static readonly List<string> ExcludedDirectories =
```

Comment:
Semantically slightly better:
```suggestion
    private static readonly IReadOnlyList<string> ExcludedDirectories =
```

Reply:
Another thought: What about excluding the directory `IBootstrapperSettings.TempDirectory` instead?

Reply:
Is there a chance that it is something different? 🤔

Reply:
Yes. See how it is calculated. It depends on some env variables.

Reply:
I will need to carry `IBoostrapperSettings` all the way to the `AdditionalFilesService` and I don't think it is worth it.
As I am testing for directory _parts_ to be `.sonarqube`, and it is [always ](https://github.com/SonarSource/sonar-scanner-msbuild/blob/657e8c48a9433b4ae3617fb0d8571d4636296b6d/src/SonarScanner.MSBuild/BootstrapperSettings.cs#L106)part of the TempDir, I think the implementation is sufficient.
Let me know what you think.